config.def.h

```diff
@@ -173,6 +173,8 @@
 
 #define DEFAULT_USER_LANGUAGE 0
 
+#define DEFAULT_LINUX_GAMEMODE_ENABLE true
```

Comment:
Not sure if we should enable GameMode by default. While it shouldn't have an impact on user experience, I think having the user opt-in may be the right move. Don't want to do something that they don't expect.

``` suggestion
#define DEFAULT_LINUX_GAMEMODE_ENABLE false
```

Reply:
I think it should be on by default. This is how Windows handles its own game mode (on unless a game is blacklisted or not recognized as a game.) It's also what games that support GameMode do. If available, they enable it. And I don't think they even offer an option to disable.

I'd say that if a user installs GameMode, they want it used plug&play style without having to go over every single game they have and explicitly enable it, which would be kind of annoying :-)